pkg/runctl/controller.go

```diff
@@ -494,6 +509,7 @@ func (c *Controller) ensurePipelineRunsConfig(ctx context.Context, pipelineRun k
 			return nil, true, err
 		}
 	}
+	logger.V(3).Info("Loaded pipeline run config")
```

Comment:
I would like to know if we want to log [pipeline run configuration (selected)](https://github.com/SAP/stewardci-core/blob/07a9665b4f0589a665b8272aa0058804c886b2c4/pkg/runctl/cfg/config.go#L38-L97) at this stage ?

Reply:
> I don't think we need it in the log, as we can look it up in the cluster. It would also create tons of duplicate log entries, as the config usually doesn't change.

My proposal is to add the pipeline config details log only in this log and not add into the context. In this way it will not create duplicate log entries but only a single entry.